packages/charts/chart-web-components/src/horizontal-bar-chart/horizontal-bar-chart.ts

```diff
@@ -71,8 +73,68 @@ export class HorizontalBarChart extends FASTElement {
     this.hydrateLegends();
   }
 
+  public enableTabbability() {
+    requestAnimationFrame(() => {
+      const legendContainer = this.shadowRoot?.getElementById('legend-div') as HTMLElement;
+      if (legendContainer) {
+        const legendButtons = tabbable.tabbable(legendContainer);
```

Comment:
do we really need tabbable for this? we already know which elements are focusable and we can select them easily.
you can refer to the implementations of the MenuList, RadioGroup, and Tablist web components to see how they handle this.

Reply:
yes, can select those elements directly also. That what is the functionality of Tabbale. It only helps in identifying the focusable elements.